docker/generic/start_proxy.py

```diff
@@ -605,6 +621,8 @@ def enforce_conflict_args(args):
         return "Flag --ssl_port is going to be deprecated, please use --ssl_server_cert_path only."
     if args.tls_mutual_auth and args.ssl_client_cert_path:
         return "Flag --tls_mutual_auth is going to be deprecated, please use --ssl_client_cert_path only."
+    if args.ssl_client_root_certs_file and args.enable_grpc_backend_ssl and args.grpc_backend_ssl_root_certs_file:
```

Comment:
This is non-empty by default so maybe we don't need put in the condition.

Reply:
Done